soh/src/code/z_parameter.c

```diff
@@ -2916,7 +2916,7 @@ void Interface_DrawItemButtons(GlobalContext* globalCtx) {
             }
 
             const s16 rCUpBtnX  = OTRGetRectDimensionFromRightEdge(R_C_UP_BTN_X+Right_HUD_Margin);
-            const s16 rCUPIconX = OTRGetRectDimensionFromRightEdge(R_C_UP_ICON_X+Right_HUD_Margin);
+            const s16 rCUPIconX = OTRGetRectDimensionFromRightEdge(R_C_UP_ICON_X+Right_HUD_Margin-!!CVar_GetS32("gNaviTextFix", 0));
```

Comment:
Why is this negated twice?

Reply:
Makes sure the value is either `0` or `1` and not something else

Reply:
Thankyou